internal/consensus/reactor.go

```diff
@@ -47,9 +47,10 @@ type Reactor struct {
 	waitSync atomic.Bool
 	eventBus *types.EventBus
 
-	rsMtx         cmtsync.RWMutex
-	rs            *cstypes.RoundState
-	initialHeight int64 // under rsMtx
+	initialHeight atomic.Int64
+
+	rsMtx cmtsync.RWMutex
+	rs    cstypes.RoundState
```

Comment:
Not sure about the benefits of this change.

Reply:
🙃 swapped order of lines in the course of doing this change and made it into the diff. I removed this superfluous change